src/components/Login.js

```diff
@@ -3,11 +3,13 @@ import { notify } from 'react-notify-toast';
 import { connect } from 'react-redux';
 import { Link, withRouter } from 'react-router-dom';
 import { LoginAction } from '../Actions/AuthActions';
+import Loader from './Loader'
 
 export class Login extends Component {
   state = {
     username: '',
     token: '',
+    loading: false
```

Comment:
lint here, arrange these keys in alphabetical order.

Reply:
Resolved thanks